static/js/components/dashboard/CourseDescription.js

```diff
@@ -130,19 +145,12 @@ export default class CourseDescription extends React.Component {
   render() {
     const { courseRun, courseTitle } = this.props;
 
-    let detailContents;
-    if (courseRun && !_.isEmpty(courseRun)) {
-      detailContents = this.renderDetailContents(courseRun);
-    } else {
-      detailContents = <span className="no-runs">No future courses are currently scheduled.</span>;
-    }
-
```

Comment:
I just moved to `renderCourseDateMessage` to make the course details more consistent, it is easier to test that way. Otherwise, I had to modify every single test for this component.